.github/workflows/check-catalyst.yaml

```
@@ -336,14 +365,17 @@ jobs:
     name: Frontend Tests
     needs: [constants, llvm, runtime, quantum]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: ${{ fromJson(needs.constants.outputs.compilers) }}
 
     steps:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v3
 
     - name: Install Deps
       run: |
-        sudo apt-get install -y python3 python3-pip libomp-dev
+        sudo apt-get install -y python3 python3-pip libomp-dev libasan5
```

To make it compatible with gcc11,
```suggestion
        sudo apt-get install -y python3 python3-pip libomp-dev libasan6
```